src/autorest-core/lib/outstanding-task-awaiter.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { OutstandingTaskAlreadyCompletedException } from './exception';
+import { OutstandingTaskAlreadyCompletedException } from "./exception";
```

Comment:
Out of curiosity, why did we have to rewrite our own versions of locking and stuff, pretty sure node has a bunch of reusable libraries